src/main/java/com/zendesk/maxwell/replication/BinlogConnectorLifecycleListener.java

```diff
@@ -0,0 +1,38 @@
+package com.zendesk.maxwell.replication;
+
+import com.github.shyiko.mysql.binlog.BinaryLogClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+class BinlogConnectorLifecycleListener implements BinaryLogClient.LifecycleListener {
+	private static final Logger LOGGER = LoggerFactory.getLogger(BinlogConnectorLifecycleListener.class);
+
+	protected final AtomicBoolean mustStop = new AtomicBoolean(false);
+
+	public void stop() {
+		mustStop.set(true);
+	}
```

Comment:
Don't think we need this.

Reply:
This method is used [here](https://github.com/zendesk/maxwell/pull/790/files#diff-6c0364725d8f72f2d70f68c6c6115747R128).

☝️  Can the usage be safely remove?

Reply:
Yup, this listener doesn't need to deal with `stop` request .